libafl/src/stages/dump.rs

```diff
@@ -138,37 +202,29 @@ where
         while let Some(i) = corpus_id {
             let mut testcase = state.corpus().get(i)?.borrow_mut();
             state.corpus().load_input_into(&mut testcase)?;
-            let bytes = (self.to_bytes)(testcase.input().as_ref().unwrap(), state);
-
-            let fname = self.corpus_dir.join(format!(
-                "id_{i}_{}",
-                testcase
-                    .filename()
-                    .as_ref()
-                    .map_or_else(|| "unnamed", String::as_str)
-            ));
+            let bytes = (self.to_bytes)(&testcase, state);
```

Comment:
I wonder if this could/should use an `InputConverter`? Or I guess it's possible to call it from the closure..?

Reply:
I would like to be able to work with testcases, not inputs, and I don't think there's an equivalent for those, is there?

Also: you probably could just call it in the closure.